ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  takeLeftToolbarScreenshot,
+  TopPanelButton,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
```

Comment:
Can we replace it with `waitForPageInit`?

Reply:
Done:)